### PR TITLE
Ins 912 display nearest well

### PR DESCRIPTION
**Pull request by AmirShayegh**

### CODE REVIEW

File: app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts [hunk 1]
```diff
@@ -136,28 +137,31 @@ export class LocationInputComponent implements OnInit {
   }
 
   get latitudeField(): FormConfigField {
-    if (this.object.isSpaceGeom) {
+    if (this.object.isSpaceGeom && this.object.isSpaceGeom.embeddedFields) {
```
Comment (fwpushan) [inline app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts]: embeddedFields are prop of object not isSpaceGeom, that is boolean

File: app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts [hunk 2]
```diff
@@ -136,28 +137,31 @@ export class LocationInputComponent implements OnInit {
...
+    if (this.fieldObject === undefined) {
+      return this.formService.getEmptyConfigField();
+    }
     const geo = this.fieldObject.geometry || this.formService.getEmptyConfigField();
     return geo;
   }
 
   get longitudeField(): FormConfigField {
-    if (this.object.isSpaceGeom) {
+    if (this.object.isSpaceGeom && this.object.isSpaceGeom.embeddedFields) {
```
Comment (fwpushan) [inline app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts]: embeddedFields are prop of object not isSpaceGeom, that is boolean

**Review by fwpushan — COMMENTED** [review]
Please

File: app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts (hunk 2 above)
Comment (AmirShayegh) [reply app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts]: fixed

File: app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts (hunk 1 above)
Comment (AmirShayegh) [reply app/lucy/src/app/components/base-form/location-input/location-input/location-input.component.ts]: fixed

**Review by fwpushan — APPROVED** [review]

**Pull request merged**